calico-enterprise/_includes/components/ReqsSys.js

```diff
@@ -29,9 +30,9 @@ function NodeRequirementsEnt(props) {
           <ul>
             {(props.orch === orchestrators.Kubernetes || props.orch === orchestrators.HostProtection) && (
               <>
-                <li>CentOS 8</li>
+                <li>CentOS Stream 8</li>
```

Comment:
If we callout CentOS stream here but the SELinux packages don't support CentOS stream. The Calico Enterprise installation will still fail as it is also a selinux-enabled system. Shall we say CentOS Stream is a supported platform and listed here?

Reply:
@tigeraphil @sujeet-kr ?